src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.block {
+  width: 200px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+
+.photo {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.discription {
+  margin: 36px 16px 0;
+  font-size: 12px;
+  font-weight: 500;
+  width: 166px;
+  height: 36px;
+  align-items: center;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.code {
+  font-size: 10px;
+  margin: 4px 0 0 16px;
+  color: #616070;
+
+}
+
+.reviews {
+  margin-right: 16px;
+  font-size: 10px;
+  color: #060b35;
+  line-height: 16px;
+}
+
+.rev {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 18px;
+}
+
+.star {
+  display: flex;
+  margin-left: 16px;
+  margin-bottom: 1px;
+
+}
+
+.stars {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star .stars:nth-child(-n+4) {
```

Comment:
Add style to all cases (from 1 to 5 stars).

Reply:
after assigning a color to each star, the tests fail. In the test, the last star is not marked with a color. How to solve this error?

> Add style to all cases (from 1 to 5 stars).

Reply:
According to the task:
![image](https://user-images.githubusercontent.com/80700247/224335288-d74d7f84-1e90-4233-8bd9-7e9f6a537d69.png)

So you need to add smth like below: 

.stars--0 > .stars__star:nth-child(0),
.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}